### DISCUSSION TITLE
Linux Binaries for Release v0.41.0

### DISCUSSION THREAD

**Discussion by robk1234** [open-telemetry/opentelemetry-collector-contrib#6950] [Q&A] [answered]
In previous releases, there were binaries for several operating systems released as assets in addition to the source code.  For v0.41.0, only the source code is released as assets.  Are binaries going to be released for future releases or only the source code?

**Comment by jpkrohling** [comment, marked as answer]
Binaries are indeed going to be generated for future releases. For more information on this specific issue, see #6714.
